components/NavBar.tsx

```diff
@@ -0,0 +1,50 @@
+import Link from 'next/link';
+import React from 'react'
+
+interface ActiveOption {
+    activeOption?: "About" | "Events" | "Gallery" | "Branches" | "Contact Us" | "CodeRED" | "Register";
+}
+const NavBar = () => {
+
+    return (
+      <nav className="flex flex-row justify-between mx-8 mt-2 text-center">
+        <div className="flex justify-between gap-4">
+            <Link href="/">
+                <img
+                src="/icons/logo.svg"
+                alt="Logo"
+                className="mx-auto"
+                />
+            </Link>
+            <h1 className="my-1 font-extrabold text-2xl text-red-700">CougarCS</h1>
+        </div>
+        <ul className="flex gap-4 px-8 mt-4">
+          <li>
+            <Link href="/">About</Link>
+          </li>
+          <li>
+            <Link href="/events">Events</Link>
+          </li>
+          <li>
+            <Link href="/gallery">Gallery</Link>
+          </li>
+          <li>
+            <Link href="/branches">Branches</Link>
+          </li>
+          <li>
+            <Link href="/contact-us">Contact Us</Link>
+          </li>
+          <li>
+            <Link href="/code-red">CodeRED</Link>
+          </li>
+            <li className="">
+                <Link className="bg-red-700 text-white px-4 py-2" href="/register">Register</Link>
```

Comment:
**fix:**  Red button only appears on hover

![image](https://github.com/CougarCS/CougarCS-Web/assets/80173797/06e93ba8-48bb-4a51-a59e-22ad08477d4d)

Reply:
Unsure of this, local looks like this 
![image](https://github.com/CougarCS/CougarCS-Web/assets/45112852/2d8312e7-5188-42d8-878e-c0763aaeaf18)

Reply:
This is my bad, I forgot to remove cache (`/.next`) - disregard this comment